lib/rucio/core/authentication.py

```diff
@@ -454,16 +455,18 @@ def validate_auth_token(token, *, session: "Session"):
                            lifetime: <token lifetime>,
                            audience: <audience>,
                            authz_scope: <authz_scope> }
-              if successful, None otherwise.
+              if successful
+    :raises: CannotAuthenticate if unsuccessful
     """
     if not token:
-        return None
+        raise CannotAuthenticate("No token was passed!")
 
     # Be gentle with bash variables, there can be whitespace
     token = token.strip()
     cache_key = token.replace(' ', '')
 
     # Check if token ca be found in cache region
+    value: "Dict[str, Any]"
     value = TOKENREGION.get(cache_key)
```

Comment:
Any reason why this is split into two lines?

Reply:
I am no longer sure why I initially handled it in this way. I changed it.